MIGRATING.md

```diff
@@ -1,5 +1,16 @@
 # Migration Guide
 
+## Migrating from versions < 14.5.0
+- Changes to `StripeIntent`
+    - `redirectData` is now deprecated.
```

Comment:
can you add a code example? (before vs after)

Reply:
Should we even be providing code samples for code no one should ever write? I guess it would more clearly show what changed